### PR TITLE
Add "product-version" command

### PR DESCRIPTION
**Pull request by xiangge**
Add "product-version" command.

JIRA: PDC-1912

### CODE REVIEW

**Comment by simozhan** [comment]
LGTM :+1:

File: pdc_client/plugins/product_version.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        fmt = '{0:20} {1}'
+        for key, value in self.prep_for_print(product_version).items():
+            print(fmt.format(key, value))
+
+    def product_version_list(self, args):
+        filters = {}
+        if args.inactive:
+            filters['active'] = False
+        elif not args.all:
+            filters['active'] = True
```
Comment (hluk) [inline pdc_client/plugins/product_version.py:95]:
```py
filters = { 'active': not args.inactive and args.all }
```

**Review by hluk — COMMENTED** [review]
+1

File: pdc_client/plugins/product_version.py (hunk 1 above)
Comment (xiangge) [reply pdc_client/plugins/product_version.py:95]: I think both the two ways are okay, so no need change the original codes, what's your thinking?

Comment (hluk) [reply pdc_client/plugins/product_version.py:95]: Yes, it's OK to keep it as it is.

**Review by hluk — APPROVED** [review]

File: pdc_client/plugins/product_version.py (hunk 1 above)
Comment (xiangge) [reply pdc_client/plugins/product_version.py:95]: Cool!

**Comment by xiangge** [comment]
Close this as patches have been merged.

**Pull request closed without merging**